pkg/transforms/transformer.go

```diff
@@ -92,6 +119,14 @@ func NewNodeEvent(event *Event, trans Transform, resourceString string) NodeEven
 		ComputeEdges: trans.BuildEdges,
 	}
 	ne.ResourceString = resourceString
+
+	// Kubernetes 1.20 deprecates selfLink. This workaround builds a self link to satisfy the existing logic.
+	// A permanent fix is delivered in ACM 2.2
+	if ne.Node.Properties["selfLink"] == nil || ne.Node.Properties["selfLink"].(string) == "" {
+		ne.Node.Properties["selfLink"] = buildSelfLink(ne.Node.Properties, resourceString)
+		fmt.Println("replacing selfLink")
```

Comment:
Remove the comment or replace with glog, if we are keeping it.

Reply:
Removed, this was for my debugging.